benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -448,17 +449,17 @@ msgstr "Select a benefit option"
 
 msgid "eligibility.pages.index.headline"
 msgstr ""
-"Connect your bank card to your public transit discount with Cal-ITP Benefits"
+"Connect your bank card to your public transit discount with Cal-ITP Benefits."
 
 #, python-format
 msgid "eligibility.pages.index.p[0]%(info_link)s"
 msgstr ""
-"You can tap your credit or debit card when you board, and your discount will "
-"automatically apply every time you ride. Cal-ITP doesn’t save any of your "
-"information, and you don’t need to create an account. Verify for your "
-"benefit, and connect your bank card today. <strong><a class='info-link' href="
-"\"%(info_link)s\">Learn more about Cal-ITP Benefits</a></strong><span "
-"class='period'>.</span>"
+"You can tap your credit or debit card when you board an MST bus, and your "
```

Comment:
Fine for this round, but we probably want to parameterize this and/or store it on the model to support additional agencies.

Reply:
Yeah this should be a ticket for the next-agency milestone.

Reply:
https://github.com/cal-itp/benefits/issues/1106